db/migrate/20191215104806_create_insights_hits.foreman_inventory_upload.rb

```diff
@@ -0,0 +1,14 @@
+class CreateInsightsHits < ActiveRecord::Migration[5.2]
+  def change
+    create_table :insights_hits do |t|
+      t.references :host, foreign_key: true
```

Comment:
Shouldn't this reference the insights facet instead of the host itself? In any way, I'd add `null: false`

Reply:
Since I need to match between host names in foreman to hostnames that come from Insights, it was easier not to query extra ID.

I have created a connection through `host_id` column on hit and on facet. Since the facet is 1:1 to host, it would work as expected.